.vscode/launch.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "ruby_lsp",
+      "request": "launch",
+      "name": "Debug test file",
+      "program": "ruby -Itest ${relativeFile}",
+    },
+  ]
+}
```

Comment:
Same as for the other file

Reply:
This files allows you to run a test with the [standard Ruby debugger](https://stdgems.org/debug/) plugged, so that you can set breakpoints from the editor and debug/inspect there.